src/main/java/io/jenkins/pluginhealth/scoring/probes/SCMLinkValidationProbe.java

```diff
@@ -65,7 +66,10 @@ public SCMLinkValidationProbe(HttpClient httpClient, GithubConfiguration githubC
     }
 
     @Override
-    public ProbeResult doApply(Plugin plugin, ProbeContext context) {
+    public ProbeResult doApply(Plugin plugin, ProbeContext context) {        final ProbeResult deprecatedProbeResult = plugin.getDetails().get(DeprecatedPluginProbe.KEY);
+        if (deprecatedProbeResult != null && deprecatedProbeResult.status().equals(ResultStatus.FAILURE)) {
```

Comment:
```suggestion
    public ProbeResult doApply(Plugin plugin, ProbeContext context) {
        final ProbeResult deprecatedProbeResult = plugin.getDetails().get(DeprecatedPluginProbe.KEY);
        if (deprecatedProbeResult != null && deprecatedProbeResult.status().equals(ResultStatus.FAILURE)) {
```
Missing line return.